extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java

```diff
@@ -134,7 +134,7 @@ public void load(String value) throws StreamLoadException {
 
     private LoadResponse loadBatch(String value) {
         Calendar calendar = Calendar.getInstance();
-        String label = String.format("audit_%s%02d%02d_%02d%02d%02d_%s",
+        String label = String.format("doris_sink_%s%02d%02d_%02d%02d%02d_%s",
```

Comment:
It is better to have prefixes similar to those related to Flink.

Reply:
It is recommended to change to doris_flink_sink_%s%02d%02d_%02d%02d%02d_%s